### ISSUE TITLE
Dev dataset not needed

### ISSUE THREAD

**Issue by NoahSchiro** [NoahSchiro/cs448_midterm#5]
Just an issue I noticed, whoever wants to take it may.

In our test train split we are also splitting out a dev dataset. The dev dataset makes sense to do when we are training over multiple epochs and want to prevent overfitting. It gives us an "runtime" estimation of the performance of the network without having to go through the whole test set. If you want more explanation of why we don't need a dev dataset, reach out to me here or comment on this issue.

So I want to cut out the dev dataset and just have train / test with an 80 / 20 split. However this is going to break a bunch of code in Logistic_reg.py and bayes.py because certain functions here are expecting there to be 3 datasets.

**Comment by NoahSchiro** [comment]
Working on this and other things in the clean-up branch
